### PR TITLE
Hello microverse project

### PR DESCRIPTION
**Pull request by Richie9000**
Added index.html, style.css, test.md, and gitignore files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="one">Hello Microverse!</header>
```
Comment (Dipeshtwis) [inline index.html]: [Optional]
- This is a good use of semantic HTML, and it'd be even better if you wrap the "Hello Microverse!" text inside a <hx> tag.
Usually the <header> tag is used as a container for the whole header of the page which is the <nav> tag.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Richie9000 , 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Your commit message is looking good.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [ ] Your pull request should have a descriptive title telling what changes are inside and also it should have a short summary describing introduced changes in more details, these make reviewing easy.
- [ ] README file should have description about the project even if it's one or two line so that anyone can understand about your project. Also it should include correct author profile link instead of someone else. Please fix that.
- [ ] `test.md` file should be in the `.gitignore` file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

__**Check my comments below**__

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Davidosky007** [comment]
Hi, @Richie9000, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
* Your pull request still doesn't have a descriptive title telling what changes are inside and also it should have a short summary describing introduced changes in more details, these make reviewing easy. please fix this and submit for review 🙏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Davidosky007** [comment]
Hi, @Richie9000, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.
# [OPTIONAL SUGGESTION]
# * Your pull request still doesn't have a descriptive title telling what changes were made with short summaries stating the changes in details, please avoid this in future projects 🙏 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**